monailabel/scribbles/transforms.py

```diff
@@ -31,6 +30,10 @@
     maxflow3d,
 )
 
+densecrf, _ = optional_import("dennseCRF")
+densecrf3d, _ = optional_import("denseCRF3D")
```

Comment:
here, you could use:
```suggestion
densecrf3d, has_densecrf3d = optional_import("denseCRF3D")
```
and then you could use `has_densecrf3d` elsewhere to display an error if the user tries to use scribbles.

Reply:
thanks! will update this.

Reply:
Looks good!